src/services/workflow.js

```diff
@@ -957,9 +974,10 @@ async function notifyAfterCompleted ({
 }) {
   const payload = {
     event: exposedEvent,
-    // if last step was successful, stopped/filtered or just skipped, this means the workflow was successful.
-    // An error interrupts the workflow.
-    status: lastLog.step.error ? 'error' : 'success',
```

Comment:
Thanks for removing this redundant property of `notification` steps after discussion on slack.

Can you make sure it’s mentioned in commit message as well?

Reply:
Yes, sure !